src/lib/process/api.ts

```diff
@@ -40,7 +41,7 @@ export const createProcessTransaction = async (
             id: data[0].toHuman(),
             version: data[1].toNumber(),
             status: 'Enabled',
-            program,
+            ...verbose ? program : {},
```

Comment:
Is there a reason we're filtering here? The concern of this function is performing the transaction, yet we're conflating that with adjusting cli output?